src/integration-test/java/com/ericsson/eiffel/remrem/generate/EiffelSemanticsController.java

```diff
@@ -58,7 +56,7 @@ public void  setUp() throws FileNotFoundException {
             then().
             statusCode(HttpStatus.SC_OK)
             .body("meta.type", Matchers.is("eiffelartifactpublished"))
-            .body("meta.version", Matchers.is("0.1.5"));
+            .body("meta.version", Matchers.is("0.1.7"));
```

Comment:
no. but we can read it from manifest file of semantics. I will update it tomorrow